### PR TITLE
PNDA-4530: Refactored pnda_env_example.yaml to support openstack heat backend.

### PR DESCRIPTION
**Pull request by AbhilashS-MapleLabs**
PNDA-4530

Factored out the pnda_env to include sections specific to cloud infra type.
Note: This is the first part of integrating OpenStack Heat backend to "pnda-cli"  repo.

Validation carried to verify the functionality of AWS after changes:
1. RHEL pico and standard for HDP for Create.

### CODE REVIEW

File: pnda_env_example.yaml [hunk 1]
```diff
@@ -1,4 +1,22 @@
-ec2_access:
+cloud_infrastructure :
```
Comment (jeclarke) [inline pnda_env_example.yaml]: I think this should be 'infrastructure' without the 'cloud' as existing-machines may not be cloud based.

File: pnda_env_example.yaml [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+  # cloud infrastructuer used for pnda deployment
```
Comment (jeclarke) [inline pnda_env_example.yaml]: typo infrastructuer

File: pnda_env_example.yaml [hunk 3]
```diff
@@ -45,6 +52,61 @@ cloud_formation_parameters:
...
+  privateSubnetGateway: 10.0.1.1
+
+  # CIDR specifying the address range for the public producer subnet (Kafka ingress)
+  publicProducerSubnetCidr: 10.0.2.0/24
+
+  # Address of gateway for Kakfa interface on public producer subnet
+  publicProducerSubnetGateway: 10.0.2.1
+
+existing_machines_parameters:
+  #Below 2 variables used in package_install.sh for offline mode
```
Comment (jeclarke) [inline pnda_env_example.yaml:99]: package-install.sh uses ${vpcCidr}, I don't see privateSubnetCidr or publicProducerSubnetCidr in there.

**Review by jeclarke — CHANGES_REQUESTED** [review]

**Comment by jeclarke** [comment]
Also please rebase this carefully on develop, we removed support for ubuntu and this will need to build on top of that without regressing any of those changes. thanks!

File: pnda_env_example.yaml (hunk 1 above)
Comment (janselva) [reply pnda_env_example.yaml]: Sure James , will change it

**Comment by jeclarke** [comment]
Please review the contribution guidelines: https://cwiki.pnda.io/display/public/PNDA/Submission+Guidelines

In particular:
"When ready, rebase your branch in interactive mode and squash all the commits to one commit"

**Comment by AbhilashS-MapleLabs** [comment]
Sorry, it was a mistake, instead of checking-in clone branch got checked in this.
Will squash all commits into one when ready.

**Comment by AbhilashS-MapleLabs** [comment]
Have rebased and addressed the request:

1. Changed the “cloud infrastructure” into infrastructure
2. Added new section in pnda_env for existing machine to support Offline mode : 
a. vpcCidr is applicable only for AWS because it’s under AWS section
b. for OpenStack and “existing machine” flavor, need to add PNDA and ingest network.

**Comment by jeclarke** [comment]
Thanks for the updates, merged now.

**Pull request merged**